scripts/learn.py

```diff
@@ -382,6 +391,7 @@ def main():
 
         # HACK: To make sure all subjects in the volume_manager are used in a batch, we have to split the trainset/validset in 2 volume managers
         model.volume_manager = validset_volume_manager
+        model.dropout_prob = 0.  # Do not use dropout for evaluation
```

Comment:
We still need to scale the activations by the dropping rate at test time. The alternative would be to use inverted dropout
https://www.quora.com/What-is-inverted-dropout
http://stats.stackexchange.com/questions/205932/dropout-scaling-the-activation-versus-inverting-the-dropout